src/main/kotlin/de/amplimind/codingchallenge/model/SubmissionStates.kt

```diff
@@ -1,12 +1,23 @@
 package de.amplimind.codingchallenge.model
 
+/**
+ * The states a submission can be in
+ */
 enum class SubmissionStates {
     /**
      * The user has not even requested his submission
      */
     INIT,
+
+    // Will be set by the user when he fetches his project for the first time
     IN_IMPLEMENTATION,
+
+    // Will be set by the user when he submits his project
     SUBMITTED,
+
+    // TODO maybe not needed
```

Comment:
Denke wenn submitted ist dann kann man davon ausgehen das es eh schon in review ist

Reply:
Jo, dann haun wirs raus^^